### PR TITLE
Lo 13 create form for address

### PR DESCRIPTION
**Pull request by RgarcesGit**
Address view

### CODE REVIEW

**Review by yoyi1073 — APPROVED** [review]

File: resources/views/address.blade.php [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        <input type="text" id="street" name="street" placeholder="Calle" required>
+    </div>
+    <br>
+    <div>
+        <label for="number">Número:</label>
+        <input type="text" id="number" name="number" placeholder="Número" required>
+    </div>
+    <br>
+    <div>
+        <label for="innernumber">Número interior:</label>
```
Comment (Diana9218) [inline resources/views/address.blade.php]: In the "for" you must put the words separated by a hyphen like:
inner-number

File: resources/views/address.blade.php [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        <input type="text" id="innernumber" name="innernumber" placeholder="Número interior" required>
```
Comment (Diana9218) [inline resources/views/address.blade.php]: The same as above, but for "id" and "name" :
inner-number

File: resources/views/address.blade.php [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    </div>
+    <br>
+    <div>
+        <label for="delg">Delegación o Municipalidad:</label>
+        <input type="text" id="delg" name="delg" placeholder="Delegación o Municipalidad" required>
+    </div>
+    <br>
+    <div>
+        <label for="delg">Delegación o Ciudad:</label>
+        <input type="text" id="delg" name="delg" placeholder="Delegación o Ciudad" required>
```
Comment (zahirolamadrid) [inline resources/views/address.blade.php]: Use city in this input

File: resources/views/address.blade.php [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    </div>
+    <br>
+    <div>
+        <label for="colony">Colonia o Municipalidad:</label>
+        <input type="text" id="colony" name="colony" placeholder="Colonia o Municipalidad" required>
+    </div>
+    <br>
+    <div>
+        <label for="delg">Delegación o Municipalidad:</label>
+        <input type="text" id="delg" name="delg" placeholder="Delegación o Municipalidad" required>
```
Comment (zahirolamadrid) [inline resources/views/address.blade.php]: Name the attributes completely

File: resources/views/address.blade.php (hunk 2 above)
Comment (zahirolamadrid) [inline resources/views/address.blade.php]: I suggest interior number and this input is not required

File: resources/views/address.blade.php (hunk 1 above)
Comment (RgarcesGit) [reply resources/views/address.blade.php]: Done

File: resources/views/address.blade.php (hunk 2 above)
Comment (RgarcesGit) [reply resources/views/address.blade.php]: Done

File: resources/views/address.blade.php (hunk 3 above)
Comment (RgarcesGit) [reply resources/views/address.blade.php]: Es en el for donde tu recomiendas poner "city"

File: resources/views/address.blade.php (hunk 4 above)
Comment (RgarcesGit) [reply resources/views/address.blade.php]: No entiendo

Comment (zahirolamadrid) [reply resources/views/address.blade.php]: image

File: resources/views/address.blade.php (hunk 3 above)
Comment (zahirolamadrid) [reply resources/views/address.blade.php]: Yes, in this input only put city

File: resources/views/address.blade.php (hunk 4 above)
Comment (zahirolamadrid) [reply resources/views/address.blade.php]: Use municipality that is a complete name. delg or munic are incomplete names.

**Review by Diana9218 — APPROVED** [review]

File: resources/views/address.blade.php (hunk 3 above)
Comment (RgarcesGit) [reply resources/views/address.blade.php]: Done

File: resources/views/address.blade.php (hunk 4 above)
Comment (RgarcesGit) [reply resources/views/address.blade.php]: Done

File: resources/views/address.blade.php (hunk 2 above)
Comment (RgarcesGit) [reply resources/views/address.blade.php]: Done

File: resources/views/address.blade.php [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+    </div>
+        <br>
+    <div>
+        <label for="number">Número:</label>
+        <input type="text" id="number" name="number" placeholder="Número" required>
+    </div>
+        <br>
+    <div>
+        <label for="inner-number">Número interior:</label>
+        <input type="text" id="inner-number" name="inner-number" placeholder="Número interior" required>
```
Comment (zahirolamadrid) [inline resources/views/address.blade.php]: I suggest interior-number in this input

File: resources/views/address.blade.php [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+    </div>
+        <br>
+    <div>
+        <label for="colony">Colonia:</label>
+        <input type="text" id="colony" name="colony" placeholder="Colonia" required>
+    </div>
+        <br>
+        <div>
+        <label for="Municipality">Delegación o Municipio:</label>
+        <input type="text" id="Municipality" name="Municipality" placeholder="Delegación o Municipio" required>
```
Comment (zahirolamadrid) [inline resources/views/address.blade.php]: Use lowercase for attribute names

File: resources/views/address.blade.php [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+    </div>
+        <br>
+    <div>
+        <label for="state">Estado:</label>
+        <input type="text" id="state" name="state" placeholder="Estado" required>
+    </div>
+        <br>
+        <div>
+            <label for="city">Ciudad:</label>
+            <input type="text" id="city" name="state" placeholder="Ciudad" required>
```
Comment (zahirolamadrid) [inline resources/views/address.blade.php]: Put city before state

File: resources/views/address.blade.php [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+        </div>
+        <br>
+    <div>
+        <label for="postcode">Código Postal:</label>
+        <input type="text" id="postcode" name="postcode" placeholder="Código Postal" required>
```
Comment (zahirolamadrid) [inline resources/views/address.blade.php]: This input is type number

File: resources/views/address.blade.php (hunk 2 above)
Comment (zahirolamadrid) [reply resources/views/address.blade.php]: I suggest interior-number in this input

File: resources/views/address.blade.php (hunk 6 above)
Comment (RgarcesGit) [reply resources/views/address.blade.php]: Done

File: resources/views/address.blade.php (hunk 5 above)
Comment (RgarcesGit) [reply resources/views/address.blade.php]: Done

File: resources/views/address.blade.php (hunk 7 above)
Comment (RgarcesGit) [reply resources/views/address.blade.php]: Done

File: resources/views/address.blade.php [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+    </div>
+        <br>
+    <div>
+        <label for="number">Número:</label>
+        <input type="text" id="number" name="number" placeholder="Número" required>
+    </div>
+        <br>
+    <div>
+        <label for="interior-number">Número interior:</label>
+        <input type="text" id="interior-number" name="interior-number" placeholder="Número interior" required>
```
Comment (zahirolamadrid) [inline resources/views/address.blade.php]: This input is not required

**Review by zahirolamadrid — APPROVED** [review]

**Pull request merged**